.github/workflows/build.yml

```
@@ -12,26 +12,30 @@ jobs:
       matrix:
         ruby: ['2.7', '3.0', '3.1', '3.2', '3.3', '3.4']
     steps:
-    - uses: actions/checkout@v4
-      with:
-        ref: ${{ github.event.pull_request.head.sha }}
-    - name: Set up Ruby ${{ matrix.ruby }}
-      uses: ruby/setup-ruby@v1
-      with:
-        ruby-version: ${{ matrix.ruby }}
-        bundler-cache: true
-    - name: Bundle install
-      run: bundle install
-    - name: Specs with coverage
-      if: ${{ matrix.ruby == 2.7 }}
-      run: "RUBYOPT='--enable=frozen-string-literal --debug=frozen-string-literal' bundle exec rake"
-      env:
-        COVERALLS_REPO_TOKEN: ${{ secrets.COVERALLS_REPO_TOKEN }}
-        CI_PULL_REQUEST: ${{ github.event.number }}
-        CI_BRANCH: ${{ github.head_ref }}
-    - name: Specs
-      if: ${{ matrix.ruby != 2.7 }}
-      run: "RUBYOPT='--enable=frozen-string-literal --debug=frozen-string-literal' bundle exec rake"
+      - uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+      - name: Set up Ruby ${{ matrix.ruby }}
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+          bundler-cache: true
+      - name: Bundle install
+        run: bundle install
+      - name: Specs with coverage
+        if: ${{ matrix.ruby == 2.7 }}
+        run: "RUBYOPT='--enable=frozen-string-literal --debug=frozen-string-literal' bundle exec rake"
+        env:
+          COVERAGE: true
+      - name: Specs
+        if: ${{ matrix.ruby != 2.7 }}
+        run: "RUBYOPT='--enable=frozen-string-literal --debug=frozen-string-literal' bundle exec rake"
+      - name: Coverage report - Ruby ${{ matrix.ruby == 2.7 }}
+        if: ${{ matrix.ruby == 2.7 }}
+        uses: joshmfrankel/simplecov-check-action@main
```

Would it be better to lock the version? Assuming we can use the tag:
```suggestion
        uses: joshmfrankel/simplecov-check-action@1.0.0
```

---

The maintainer doesnt do release anymore last one was 2022. The example in the readme use `main`. I can lock it to a commit :)